force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js

```diff
@@ -19,77 +20,25 @@ export default class RecurringDonationTable extends LightningElement {
 
     lastDonationDate = '';
     
-    data = [{
-        "id": 1,
-        "firstDonation": "7/4/2022",
-        "donor": "Saba",
-        "amount": 69,
-        "paymentMethod": "Credit Card, *1111Exp. 11/2023",
-        "frequency": "Monthly on the 22nd",
-        "status": "Active",
-        "mostRecenDonation": "12/10/2023",
-        "lastModified": "4/01/2023",
-        "nextDonation": "Paid Donation: 1 of 5. Next Donation: 1/12/2022."
-      },
-      {
-        "id": 2,
-        "firstDonation": "4/10/2022",
-        "donor": "Siward",
-        "amount": 43,
-        "paymentMethod": "ACH, *1111",
-        "frequency": "Monthly on the 22nd",
-        "status": "Active",
-        "mostRecenDonation": "12/10/2023",
-        "lastModified": "4/01/2023",
-        "nextDonation": "Paid Donation: 3 of 6. Next Donation: 12/20/2022."
-      },
-      {
-        "id": 3,
-        "firstDonation": "3/31/2022",
-        "donor": "Page",
-        "amount": 43,
-        "paymentMethod": "Credit Card, *1111Exp. 11/2023",
-        "frequency": "Monthly on the 22nd",
-        "status": "Active",
-        "mostRecenDonation": "12/10/2023",
-        "lastModified": "4/01/2023",
-        "nextDonation": "Paid Donation: 2 of 5. Next Donation: 2/22/2022."
-      },
-      {
-        "id": 4,
-        "firstDonation": "6/25/2022",
-        "donor": "Eric",
-        "amount": 91,
-        "paymentMethod": "Check",
-        "frequency": "Monthly on the 22nd",
-        "status": "Active",
-        "mostRecenDonation": "12/10/2023",
-        "lastModified": "4/01/2023",
-        "nextDonation": "Paid Donation: 2 of 10. Next Donation: 11/11/2022."
-      },
-      {
-        "id": 5,
-        "firstDonation": "5/18/2022",
-        "donor": "Merrill",
-        "amount": 10,
-        "paymentMethod": "Credit Card",
-        "frequency": "Monthly on the 22nd",
-        "status": "Active",
-        "mostRecenDonation": "12/10/2023",
-        "lastModified": "4/01/2023",
-        "nextDonation": "Paid Donation: 6 of 8. Next Donation: 12/22/2022."
-    }];
+    data = [];
+
     actions = [
       { label: 'View', name: 'view' },
       { label: 'Delete', name: 'delete' }
     ];
+
     columns = [];
+
+	connectedCallback() {
```

Comment:
Since we can not determine if @wire method or connectedCallback method will run first, maybe we could remove this connectedCallback() method and call this.retrieveData() from inside the @wire method? I recall we have done this some times in the past...